src/transformStateWithClones.js

```diff
@@ -6,8 +6,46 @@
  *
  * @return {Object[]}
  */
-function transformStateWithClones(state, actions) {
-  // write code here
+function transformState(state, actions) {
+  const result = [];
+  const copyState = { ...state };
+
+  for (const [key, value] of Object.entries(copyState)) {
```

Comment:
Remove the loop that trims the values, as it is not required by the task description.